README.md

```diff
@@ -25,12 +25,55 @@ Also supports [ember-engines](https://github.com/dgeb/ember-engines)
 ember install ember-frost-navigation
 ```
 
-## Examples
-
-<img src='assets/ex1.gif'/>
+## Usage
 
 Add the `{{frost-navigation}}` component to the template where you want the navigation to appear, then configure your navigation in `app/router.js`
 
+#### Template
+```hbs
+{{frost-navigation
+  logo=(component 'custom-logo'
+    class='nav-category navigation-logo'
+  )
+  menu=(component 'user-pane'
```

Comment:
The component here is essentially `'foo'` right?  Or do we provide an actual menu component to plug in there?

If it's an actual menu component we're providing, I think we need to change the interface a bit (flatten out model, rename logo to icon)